test/test_ops.py

```diff
@@ -254,6 +257,122 @@ def test_multiple_devices(self, devices, dtype, op):
                     "Skipped! Only supports single tensor or iterable of tensor outputs."
                 )
 
+    def test_reduction_tag_coverage(self):
+        reduction_op_names = [
+            op.name for op in op_db if isinstance(op, ReductionOpInfo)
+        ]
+        reduction_op_names = [
+            op_name
+            for op_name in reduction_op_names
+            if not op_name.startswith("masked.")
+        ]
+        reduction_op_names = [
+            op_name.replace(".", "_") for op_name in reduction_op_names
+        ]  # Fix linalg.vector_norm
+
+        # Additional reduction operators
+        reduction_op_names += [
```

Comment:
nit: instead of all of the string parsing here, can we use `OpOverload` objects directly? That way you can just check `func in reduction_ops` instead of having to parse the names:
```
reduction_op_names += [
    torch.ops.aten.norm.ScalarOpt_dtype,
    torch.ops.aten.norm.Scalar,
]
```

The downside is that you need to manually write out each overload too, although I'd argue this is good since it makes things more explicit (you needed to manually ad the tag for each OpOverload anyway)